Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,11 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+var wholeNumberPart = Math.floor(num);
```

Comment:
It is better to use 'let' than 'var' to declare variable.

Reply:
I have updated